qiskit/scheduler/utils.py

```diff
@@ -1,29 +0,0 @@
-# This code is part of Qiskit.
-#
-# (C) Copyright IBM 2019.
-#
-# This code is licensed under the Apache License, Version 2.0. You may
-# obtain a copy of this license in the LICENSE.txt file in the root directory
-# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
-#
-# Any modifications or derivative works of this code must retain this
-# copyright notice, and modified files need to carry a notice indicating
-# that they have been altered from the originals.
-
-"""Scheduling utility functions."""
-
-from qiskit.utils.deprecation import deprecate_function
-from qiskit.pulse import macros, utils
-
-format_meas_map = deprecate_function(
-    '"format_meas_map" has been moved to "qiskit.pulse.utils"',
-    since="0.15.0",
-)(utils.format_meas_map)
-
-measure = deprecate_function('"measure" has been moved to "qiskit.pulse.macros"', since="0.15.0")(
-    macros.measure
-)
-
-measure_all = deprecate_function(
-    '"measure_all" has been moved to "qiskit.pulse.macros"', since="0.15.0"
-)(macros.measure_all)
```

Comment:
Probably a good thing that we're getting rid of this, because it will have been breaking the documentation of `qiskit.pulse.macros.measure_all` (etc) because `deprecate_function`/`deprecate_func` mutate their inputs.  Fortunately it looks like we don't actually seem to document those inner functions, only the `pulse.builder.measure_all` wrapper around it?

Reply:
There's nothing you need to change, it was just a comment about how this deprecation was broken by #9790, but fortunately it won't actually have leaked out to users.